src/lib/dom-api.html

```diff
@@ -593,6 +593,61 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        configurable: true,
+        get: function() {
+          var ownerDocument = this.node.ownerDocument;
+
+          // Shadow DOM
+          if (Polymer.Settings.useShadow) {
```

Comment:
This section should be its own getter in the `useShadow` section of this module.